test_dir.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
It seems `test_dir.sh` is never executed in our testing?

Reply:
No, it just updates the golden file directory.  Will rename in a followup.